mage_ai/data_preparation/templates/data_exporters/streaming/nats.yaml

```diff
@@ -0,0 +1,25 @@
+connector_type: nats
```

Comment:
You have not added the support for data exporter. Hence, this template is not required.